src/ceph_broker.py

```diff
@@ -253,6 +256,40 @@ def process_requests_v1(reqs):
                 ret = handle_erasure_pool(request=req, service=svc)
             else:
                 ret = handle_replicated_pool(request=req, service=svc)
+        elif op == "create-cephfs":
```

Comment:
NIT Suggestion:
Rather than using an if-elif-else tree to trickle down the op to it's method we could use a {'opCode': opMethod(request, service)} dict. This:
1. Provides a template for future additions to broker ops.
2. Fetches the required method in constant time rather than trickling through all the ops.

Reply:
Rewrote all this with a jump table.